src/Symfony/Component/Security/Tests/TranslationSyncStatusTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Tests;
+
+use Symfony\Component\Finder\Finder;
+
+class TranslationSyncStatusTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @dataProvider getTranslationDirectoriesData
+     */
+    public function testTranslationFileIsNotMissingInCore($dir1, $dir2)
+    {
+        $finder = new Finder();
+        $files = $finder->in($dir1)->files();
+
+        foreach ($files as $file) {
+            $filename = basename($file);
+
+            $this->assertTrue(file_exists($dir2.'/'.$filename), 'Missing file '.$filename.' in directory '.$dir2);
+        }
+    }
+
+    public function getTranslationDirectoriesData()
+    {
+        $legacyTranslationsDir = $this->getLegacyTranslationsDirectory();
+        $coreTranslationsDir = $this->getCoreTranslationsDirectory();
+
+        return array(
+            'file-not-missing-in-core' => array($legacyTranslationsDir, $coreTranslationsDir),
+            'file-not-added-in-core' => array($coreTranslationsDir, $legacyTranslationsDir),
+        );
+    }
+
+    public function testFileContentsAreEqual()
+    {
+        $finder = new Finder();
+        $files = $finder->in($this->getLegacyTranslationsDirectory())->files();
+
+        foreach ($files as $file) {
+            $coreFile = $this->getCoreTranslationsDirectory().'/'.basename($file);
+
+            $this->assertEquals(file_get_contents($file), file_get_contents($coreFile), $file.' and '.$coreFile.' have equal content.');
+        }
+    }
+
+    private function getLegacyTranslationsDirectory()
+    {
+        return __DIR__.'/../Resources/translations';
+    }
+
+    private function getCoreTranslationsDirectory()
+    {
+        $securityClass = new \ReflectionClass('Symfony\Component\Security\Core\Security');
+
+        return dirname($securityClass->getFileName()).'/Resources/translations';
```

Comment:
When testing it from the Security component, I don't think there is the necessity to find the `translation` dir like this.
It'll always be into `__DIR__.'/../Core/Resources/translations'`, right ?